native-audio/app/src/main/java/com/example/nativeaudio/NativeAudio.java

```diff
@@ -155,12 +170,19 @@ public void onClick(View view) {
 
         ((Button) findViewById(R.id.pause_uri)).setOnClickListener(new OnClickListener() {
             public void onClick(View view) {
+                if(!permissionGranted) {
+                    // Log.e(TAG, "Error: requested permission not granted, please restart app and grant access");
```

Comment:
makes sense, filed #98 so we remember to cleanup logging across all the samples later